packages/expo-face-detector/ios/EXFaceDetector.podspec

```diff
@@ -16,7 +16,14 @@ Pod::Spec.new do |s|
   s.dependency 'UMCore'
   s.dependency 'UMFaceDetectorInterface'
   s.dependency 'UMFileSystemInterface'
+
+  # even though `GoogleMLKit/FaceDetection` depends on all `MLKit*` references below
+  # framework generation code (prebuilds) cannot locate them properly, so these are defined explicitly
+  # TODO: research why xcodegen fails to detect dependencies of dependencies (resulted .xcodeproj is missing them)
   s.dependency 'GoogleMLKit/FaceDetection', '2.1.0'
+  s.dependency 'MLKitFaceDetection', '1.2.0'
+  s.dependency 'MLKitCommon', '2.1.0'
+  s.dependency 'MLKitVision', '1.2.0'
```

Comment:
is there anything we can do to avoid having to define these individual version numbers separately/explicitly? would be nice to just bump one number when upgrading, for example.

Reply:
Well, I'm not sure at this point. 🤔 
We do have this questionable mechanism when we use use this pattern: `s.dependency 'DependencyName', $SomeVersionVariable || 'x.y.z'`, but when I look on this table https://developers.google.com/ml-kit/release-notes I feel that each of these libraries is going live their own version life 🤔 (looking at Android section you'll notice that major versions may be almost the same, but minor/patch are different across libraries).
I believe that having these versions defined explicitly is not a bad approach 🤔 
After all, these occur only if a few places and if you want to bump them you'll just perform `find/replace` in Xcode/VSCode and if you missed one of them, cocoapods would hit you with an error message 😉